packages/jaeger-ui/src/reducers/index.js

```diff
@@ -16,13 +16,15 @@ import { reducer as formReducer } from 'redux-form';
 
 import config from './config';
 import dependencies from './dependencies';
+import deepDependencyGraph from './deep-dependency-graph';
 import embedded from './embedded';
 import services from './services';
 import trace from './trace';
 
 export default {
   config,
   dependencies,
+  deepDependencyGraph,
```

Comment:
Just curious, why did you decide not to use the duck approach?

Reply:
I don't love the duck approach. Separating actions and reducers makes tests easier and more atomic IMO.